libs/community/langchain_community/vectorstores/clickhouse.py

```diff
@@ -72,7 +72,7 @@ class ClickhouseSettings(BaseSettings):
     username: Optional[str] = None
     password: Optional[str] = None
 
-    index_type: str = "annoy"
+    index_type: Optional[str] = "annoy"
```

Comment:
What is this supposed to do? I think this will have unintended consequences in some string substitution stuff lower in the file